packages/twenty-server/src/engine/core-modules/file/file-upload/services/file-upload.service.ts

```diff
@@ -20,17 +20,23 @@ export class FileUploadService {
     filename,
     mimeType,
     fileFolder,
+    workspaceId,
   }: {
     file: Buffer | Uint8Array | string;
     filename: string;
     mimeType: string | undefined;
     fileFolder: FileFolder;
+    workspaceId: string | undefined;
   }) {
+    const folder = workspaceId
+      ? `${fileFolder}/workspace-${workspaceId}`
```

Comment:
Can we do `work-${workspaceId}/${fileFolder}` instead?

Reply:
(that way we can maybe share direct S3 access to `work-${workspaceId}` to workspace owners and they'll have access to all of their files)